int32 size
float32[] list_xglob
float32[] list_xcurv